css/form.css

```diff
@@ -1,15 +1,31 @@
-/* Node edit buttons */
-#edit-submit {
-    background-color: rgba(191, 213, 193, 1)
+/* Border under main webform */
+#edit-field-descriptive-metadata-wrapper {
+    border-bottom: 1px #656565 solid;
+}
+/* Buttons */
+#edit-submit, #edit-submit--2, .form-submit {
```

Comment:
Is this targeting same html id as https://github.com/mnylc/esie_subtheme/pull/9/files#diff-6a1817805a1dbb597d50598859417457R86 ?

Reply:
@DiegoPino yes you are right. I could group in a better way in same file. Assuming there will be more edits, I can push that then.